digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/AssetDataImporter.scala

```diff
@@ -1285,8 +1285,41 @@ def insertNumberPropertyData(propertyId: Long, assetId: Long, value:Int) {
       }else{
         sqlu"update text_property_value set value_fi = $vname, modified_date = sysdate, modified_by = $modifiedBy where asset_id = $assetId and property_id = $propertyVal".execute
       }
+    }
+
+  def insertNewAsset(typeId: Int, linkId: Long, startMeasure: Double, endMeasure: Double, sideCode: Int, value: Int) {
+    val assetId = Sequences.nextPrimaryKeySeqValue
+    val lrmPositionId = Sequences.nextLrmPositionPrimaryKeySeqValue
+    val propertyId = sql"""select id from property where public_id = 'mittarajoitus'""".as[Long].first
```

Comment:
On my opinion, since this affects all linear assets with numeric property value I belive this fix can be done in a differente task. Because we have to be sure to change the property id for all existing property values.